### ISSUE TITLE
Support for basepath

### ISSUE THREAD

**Issue by jeetiss** [molefrog/wouter#43]
Hello again 🙌🏻

I'm made demo on [gh-pages](https://jeetiss.github.io/wouter-async-routes/) and I can't specify base path for router (in my case it `wouter-async-routes`)

need this feature 💁🏻‍♂️

**Comment by molefrog** [comment]
If I get you right what you need is being able to match a path with a constant prefix like so:
/app/users => /users

You can easily achieve that via the custom hook feature released in v2.0.0. It allows you to specify your own `useLocation` hook which is used for obtaining the current location. I could imagine that what you need could look like:
```js
import { useLocation, Router } from "wouter";

const usePartialLocation = () => {
  const [location, setLocation] = useLocation();

  const stripped = location.replace(/^\/app/, '')
  return [stripped, to => setLocation("/app' + to)]
}
```

**Comment by jeetiss** [comment]
Nice!!!
thanks for explanation

> now [demo](https://jeetiss.github.io/wouter-async-routes/) works as expected
